x-pack/plugin/security/qa/operator-privileges-tests/src/javaRestTest/java/org/elasticsearch/xpack/security/operator/Constants.java

```diff
@@ -277,6 +277,7 @@ public class Constants {
         "cluster:monitor/ccr/follow_info",
         "cluster:monitor/ccr/follow_stats",
         "cluster:monitor/ccr/stats",
+        DataLifecycle.isEnabled() ? "cluster:monitor/xpack/usage/data_lifecycle" : null,
```

Comment:
This list is in alphabetic order. So I'd place this new entry right before `"cluster:monitor/xpack/usage/data_streams"`.

Reply:
Absolutely, I panicked after breaking main and I was sloppy. Thank you so much for being my second pair of eyes!